CONTRIBUTING.md

```diff
@@ -12,14 +12,16 @@ The main way to contribute to Resque is to write some code! Here's how:
    branch
 5. That's it!
 
-If you're not doing some sort of refactoring, a CHANGELOG entry is appropriate.
-Please include them in pull requests adding features or fixing bugs.
+An entry in the `HISTORY.md` is appropriate for changes that aren't of purely
+internal consequences. Please include a modification of HISTORY.md adding a
+bullet for your change in your pull request where relevant.
 
 Oh, and 80 character columns, please!
 
-For more information, see
-Steve Klabnik's post here about the direction and current state of Resque:
-[Rescuing Resque...again](http://words.steveklabnik.com/rescuing-resque-again)
+See Steve Klabnik's post (2015) here about the direction and state of Resque:
+[Rescuing Resque...again](http://words.steveklabnik.com/rescuing-resque-again).
+TODO: include some more up-to-date context on current state of
```

Comment:
I didn't like leaving it suggesting that a 2015 post by klabnik (who isn't involved anymore?) describes the current direction and state of resque. We know it does *not*, and I think we need to say so instead of let it imply that.  And also I like acknowledging that the current state of and direction of resque is in fact not very transparent to users, and we know that could be improved. 

Maybe there's a different way to say that then the "TODO"?  I worry just eliminating the TODO without something else leaves it seeming to tell readers that the 2015 doc by klabnick actually does describe resque current state and direction. 

But your call!

Reply:
Like honestly, for me part of it is when I see the statement "see teh state and direction of resque here" and it's from 2015 and no longer up to date -- I assume the project is abandoned!  When I see someone made a commit saying "Yeah, we know we need to update this" then I think "oh, someone is home, the lights are on!"

Obviously, actually updating it would be better. You may be the only person with the context to do so!

Reply:
Honestly, I would just remove those few lines entirely that link to Steve's post.

Reply:
OK, your call, removing it! 

Not necessary to resolve in this PR, but I do think it would be good to put somethign in the README about current status and direction of resque, as this line used to do in 2015.  As a resque user, it was difficult to figure out if it was currently maintained etc, and I spent time trying to figure out what was going on that could be saved with a little explanation!

Reply:
I think part of the issue is that a lot of the maintainers from a few years ago have moved on to working on other things, and we haven't had a lot of newer maintainers join. I don't know that there really is a direction for it at the moment. I'd be happy to bring some new maintainers on board who are eager to work on Resque.